get-involved.html

```diff
@@ -130,19 +138,29 @@
 						<span class="emphasized-header">Volunteer </span>
 					</div>
 					<div class="row">
-						<div class="col-md-6 col-sm-6 col-xs-6">
+						<div class="col-sm-6 col-xs-12">
 							<p>All of us can be seekers, searching for connection, for meaning, and for growth in our lives. All of us can support other seekers in their own journeys.</p>
 							<p>You can volunteer to help SeekHealing in its mission to facilitate the healing journey of seekers struggling with heroin or opioid addiction.</p>
 							<p>Whatever your background, wherever you are in live, reach out today to find out how you can help a fellow seeker through the SeekHealing project.</p>
 						</div>
-						<div class="col-md-6 col-sm-6 col-xs-6 text-center">
+						<div class="col-sm-6 col-xs-12 text-center">
 							<div class="row">
 								<div class="col-md-12">
-									<img src="images/owlScienceCrop.png" width="130px" height="auto" />
+									<img src="images/owl-volunteer.png" width="200px" height="auto" />
 									<a href="mailto:info@seekhealing.org?subject=Volunteer+inquiry" class="button button-3d button-rounded button-purple">Volunteer today</a>
 								</div>
 							</div>
 						</div>
+						<div class="col-xs-12 text-center volunteers">
+							<span class="subheader">CURRENTLY RECRUITING VOLUNTEERS WITH THE FOLLOWING SKILLS:</span>
+							<ul class="positions-list">
+								<li>Fundraising</li>
+								<li>Press relations</li>
+								<li>Accounting work</li>
+								<li>Grant writing</li>
+								<li>SEO & social media marketing</li>
```

Comment:
Do you mind converting this to an `&amp;`? Thanks!

Reply:
HTML Entity used now, fixed in 9030390

@j00bar ready for round 2 of review.